### PR TITLE
Empty body fix

### PR DESCRIPTION
**Pull request by itkovian**
Functions must not have an empty body?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```
Comment (boegel) [inline setup.py]: is this really worth a version bump?

the `pass` should be there, for sure, but this is just a style cleanup (functionality was not changed)

Comment (wdpypere) [inline setup.py]: if it is an rpm, it requires a bump every time. Otherwise you can't predict what is on the systems.

Comment (boegel) [inline setup.py]: Sure, but this makes NO difference in terms of functionality. So why would we roll out a new RPM specifically for this?

Comment (wdpypere) [inline setup.py]: that is another question. but I don't want 2 rpms with the same version and different checksums.

Comment (boegel) [inline setup.py]: so then make this 2.2.2dev? (which should never be rolled out as an RPM)

Comment (wdpypere) [inline setup.py]: we have no way of controlling that today. Also it does not pose that big of a problem/issue.

Comment (itkovian) [inline setup.py]: It does make a large difference. If the original function would have been called, your code would bork. Now it won't.

Comment (boegel) [inline setup.py]:
```
[kehoste:~] $ cat t.py 
def f():
    """f"""

def g():
    """g"""
    print 'in g'

g()
f()
[kehoste:~] $ python t.py
in g
```

Comment (boegel) [inline setup.py]: the `pass` is included implicitly, so this is nothing more than a style fix...

**Pull request merged**